test/net_http_test.rb

```diff
@@ -230,7 +231,8 @@ def test_use_custom_configuration_to_combine_endpoints_into_one_resource
     semian_config["development"]["nethttp_default"] = DEFAULT_SEMIAN_OPTIONS
     sample_env = "development"
 
-    semian_configuration_proc = proc do
+    semian_configuration_proc = proc do |host, port|
+      next nil if host == "127.0.0.1" && port == 8474 # # disable if toxiproxy
```

Comment:
As opposed to `next nil`? Can't, since blocks capture the stack return position (or context, or whatever you call it) as well, using `return` would return out of whatever function `yield`ed. So `next` is what it has to be. If it was a `lambda` a `return` would be fine.